### ISSUE TITLE
Cannot authenticate with Redis when a password is required.

### ISSUE THREAD

**Issue by maxolasersquad** [owncloud/core#20937]
I setup a password in my redis.conf file like this.
```
requirepass foo
```

And then in Owncloud's config.php I setup redis like so.
```
  'memcache.local' => '\\OC\\Memcache\\Redis',
  'redis' => array(
    'host' => 'localhost',
    'port' => 6379,
    'timeout' => 0.0,
    'password' => 'foo',
  ),
```

However Owncloud just gives me a white screen and I see this in my error logs.
```
2015/12/03 21:59:34 [error] 8683#0: *8438 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught RedisException: Failed to AUTH connection in /path/to/owncloud/lib/private/memcache/redis.php:74
```

Looking at redis.php there is no attempt to read the password entry or make any calls to the ```auth``` command provided by phpredis.

The documentation at https://doc.owncloud.org/server/8.2/admin_manual/configuration_files/files_locking_transactional.html recommends using a password, but this does not currently seem possible.

**Comment by maxolasersquad** [comment]
I was going to add this feature in, but I see that Lukas Reschke already did this in commit 78cad94ff4676ef401d2a75c8448f3726deefd18 Perhaps the 8.2 documentation should be updated to not recommend and show configurations for authentication.

**Comment by RealRancor** [comment]
@maxolasersquad See https://github.com/owncloud/core/issues/20652#issuecomment-158571876

Password support will be available with 8.2.2: https://github.com/owncloud/core/pull/20690
